### PR TITLE
Add a decryption converter

### PR DESCRIPTION
**Pull request by xzhang27**
Dear Gobblin maintainers,

Please accept this PR. I understand that it will not be reviewed until I have checked off all the steps below!

### JIRA
- [ ] My PR addresses the following [Gobblin JIRA](https://issues.apache.org/jira/browse/GOBBLIN/) issues and references them in the PR title. For example, "[GOBBLIN-XXX] My Gobblin PR"
    - https://issues.apache.org/jira/browse/GOBBLIN-XXX

### Description
- [ ] Here are some details about my PR, including screenshots (if applicable):

### Tests
- [ ] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:

### Commits
- [ ] My commits all reference JIRA issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
    1. Subject is separated from body by a blank line
    2. Subject is limited to 50 characters
    3. Subject does not end with a period
    4. Subject uses the imperative mood ("add", not "adding")
    5. Body wraps at 72 characters
    6. Body explains "what" and "why", not "how"

### CODE REVIEW

File: gobblin-modules/gobblin-azkaban/src/main/java/org/apache/gobblin/azkaban/AzkabanJobLauncher.java [hunk 1]
```diff
@@ -224,7 +224,7 @@ public AzkabanJobLauncher(String jobId, Properties props)
         this.closer.register(new ServiceBasedAppLauncher(jobProps, "Azkaban-" + UUID.randomUUID()));
   }
 
-  private JobListener initJobListener() {
```
Comment (eogren) [inline gobblin-modules/gobblin-azkaban/src/main/java/org/apache/gobblin/azkaban/AzkabanJobLauncher.java]: Was this change intentional?

Comment (xzhang27) [reply gobblin-modules/gobblin-azkaban/src/main/java/org/apache/gobblin/azkaban/AzkabanJobLauncher.java]: This was from the previous commit.

**Comment by eogren** [comment]
BTW, this is linked to https://issues.apache.org/jira/projects/GOBBLIN/issues/GOBBLIN-427

**Comment by htran1** [comment]
Build failure due to missing license header.
